api/compaction_manager.cc

```diff
@@ -38,6 +39,16 @@ static future<json::json_return_type> get_cm_stats(http_context& ctx,
         return make_ready_future<json::json_return_type>(res);
     });
 }
+static std::unordered_map<std::pair<sstring, sstring>, uint64_t, utils::tuple_hash> sum_pending_tasks(std::unordered_map<std::pair<sstring, sstring>, uint64_t, utils::tuple_hash> a,
+        const std::unordered_map<std::pair<sstring, sstring>, uint64_t, utils::tuple_hash>& b) {
+    for (auto&& i : b) {
+        if (i.second) {
+            a[i.first] += i.second;
```

Comment:
I think we can remove this if too, no?

Reply:
it's an optimization, this will prevent sending zeroed results (which is what most of the tables will return)

Reply:
But not if they are present in a. But I guess the first a will come from an empty map, so ok.

Reply:
exactly, I've validated that this is the case